projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -436,3 +430,63 @@ export function reviewB2bOrderConfirmation(
     cartData.totalAndShipping
   );
 }
+
+export function interceptPaymentTypesEndpoint(): string {
+  const alias = 'getPaymentTypes';
+  cy.intercept({
+    method: 'GET',
+    path: `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+      'BASE_SITE'
+    )}/paymenttypes*`,
+  }).as(alias);
+
+  return alias;
+}
+
+export function interceptCartWithDeliveryAddressEndpoint(): string {
```

Comment:
we don't need this. In checkout, all 'set' checkout details are found in the checkout details request. 

This function is is intercepting the same thing as `interceptCheckoutB2BDetailsEndpoint`, therefore we do not it.

Reply:
Yeah, that's true. Sorry, I had to miss it.